index.ch.md

```diff
@@ -0,0 +1,21 @@
+_2021-03-23_
+
+Richard M. Stallman(常被称为RMS)，很长时间以来在自由软件社区内都是一股危险的力量。他过去表现出了很多不当行为，比如歧视女性、歧视残障人士、歧视跨性别人群。这些行为不应该在自由软件、数字权利和技术社区中立足。随着他最近重新回到自由软件基金董事会，我们呼吁FSF董事会下台，并要求撤销RMS的所有领导职位。
+
+我们，作为本文件的署名人，相信数字自治的必要性，以及用户自由在保护我们的基本人权方面发挥着关键作用。为了实现软件自由使一切成为可能的承诺，必须在社区内部进行彻底的改变。我们相信，在现在和未来，所有的技术都将赋予人们力量，而不是压迫人们。我们知道，只有在一个技术建立在我们最基本的人权受到尊重的世界里，这才有可能。虽然Richard M. Stallman以某种形式推广了这些观点，但他并不能代表我们。我们不能宽恕他的行为和观点。我们不承认他或自由软件基金会今天的领导地位。
```

Comment:
```suggestion
我们，作为本文件的署名人，相信数字自治的必要性，以及用户自由在保护我们的基本人权方面发挥着的关键作用。为了实现“软件自由使一切成为可能”的承诺，必须在社区内部进行彻底的改变。我们相信，在现在和未来，所有的技术都将赋予人们力量，而不是压迫人们。我们知道，只有在一个技术从根本上尊重人权的世界里，这才有可能。虽然RMS以某种形式推广了这些观点，但他并不能代表我们。我们不能宽恕他的行为和观点。我们不承认他或自由软件基金会今天的领导地位。
```

Reply:
> We know that this is only possible in a world where technology is built to pay respect to our rights at its most foundational levels.

I suggest `at its most foundational levels` is a phrase to modify `technology`, not `our rights`.

And I think you should stick to RMS or Richard M. Stallman, either is ok, but they are not mixed together.